### ISSUE TITLE
训练出现nan现象

### ISSUE THREAD

**Issue by WoWuQ** [lc222/MPCNN-sentence-similarity-tensorflow#3]
你好，用代码跑模型时，loss一直输出nan，权重也不再调整，请问是什么问题造成的呢

**Comment by lc222** [comment]
一开始我跑这个模型的时候有出现这种情况，把学习率调小一点就行了。应该是步长过大导致的震荡

**Comment by WoWuQ** [comment]
你好，那你出现过，模型训练的时候loss不下降，一直保持一个值，也就是一直无法更新参数的现象么

**Comment by lc222** [comment]
这个到没见过，当时这个仿真最后也是有问题并没有达到论文中的效果。震荡非常严重，没有办法收敛

**Comment by WoWuQ** [comment]
好的，明白了，谢谢

**Comment by xqtbox** [comment]
请问“学习率调小”是指`tf.app.flags.DEFINE_float('lr', 1e-7, 'learning rate') # 1e-3`这一句吗？ 为什么我调小之后 loss还是nan？ 求帮助
@WoWuQ  @lc222
